addons/im_livechat/static/tests/chat_window_patch.test.js

```diff
@@ -1,31 +1,27 @@
-import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { click, contains, startClient, startServer } from "@mail/../tests/mail_test_helpers";
+import { describe, test } from "@odoo/hoot";
+import { Command, serverState } from "@web/../tests/web_test_helpers";
+import { defineLivechatModels } from "./livechat_test_helpers";
 
-import { start } from "@mail/../tests/helpers/test_utils";
+describe.current.tags("desktop");
+defineLivechatModels();
 
-import { click, contains } from "@web/../tests/utils";
-
-QUnit.module("chat window (patch)");
-
-QUnit.test("closing a chat window with no message from admin side unpins it", async (assert) => {
+test("closing a chat window with no message from admin side unpins it", async () => {
     const pyEnv = await startServer();
     const partnerId = pyEnv["res.partner"].create({ name: "Demo" });
     pyEnv["res.users"].create({ partner_id: partnerId });
     pyEnv["discuss.channel"].create({
         channel_member_ids: [
-            [
-                0,
-                0,
-                {
-                    is_pinned: true,
-                    partner_id: pyEnv.currentPartnerId,
-                },
-            ],
-            [0, 0, { partner_id: partnerId }],
+            Command.create({
+                is_pinned: true,
+                partner_id: serverState.partnerId,
+            }),
+            Command.create({ partner_id: partnerId }),
         ],
         channel_type: "livechat",
         uuid: "channel-10-uuid",
     });
-    await start();
+    await startClient();
```

Comment:
I would revert this rename, it's making way too much noise on an already big PR.
And we would need to further decide whether it's good, as it is a more complex name.

I would argue that we should not have to call `startServer` manually and it should be done automatically at the start of each test, removing the need to differentiate them.

Reply:
I still think the rename of `start` to `startClient` is good. But to not block this PR further, I will keep name `start` for the time being

Reply:
I don't think the revert to start has been applied yet?